tests/test_module.py

```diff
@@ -0,0 +1,124 @@
+"""
```

Comment:
pytest looks for modules prefixed with test_ and then runs methods prefixed with test_ as unit tests. Grouping each test within a class that relates to the class that it is testing.